net/src/mock_worker.rs

```diff
@@ -389,7 +389,7 @@ mod tests {
 
         if let ProtocolWrapper::HandleSend(msg) = res {
             cli2.receive(
-                ProtocolWrapper::HandleSendResult(MessageData {
+                ProtocolWrapper::SendResult(MessageData {
```

Comment:
@lucksus 
> ProtocolWrapper::SendResult and ProtocolWrapper::HandleSendResult were mixed up (right, @neonphog? Or did I get it wrong?) I've swapped them to have handle always mean that the network requests core to handle something

I was thinking of this the other way. When I wrote it, I was thinking the Sender would send the `SendMessage` and receive the `SendResult` and the remote node (the Handler) would receive the `HandleSend` and send the `HandleSendResult`.

Either way the names are a bit confusing... so if you have other suggestions, feel free to adjust.

Reply:
Ah, ok. Sorry. Makes sense. I think I will switch back then.